tests/run/t920.scala

```diff
@@ -1,20 +0,0 @@
-object Test {
```

Comment:
Then I don't know what it would test?


On Mon, Feb 15, 2016 at 5:40 PM, Guillaume Martres <notifications@github.com
> wrote:

> In tests/run/t920.scala
> <https://github.com/lampepfl/dotty/pull/1082#discussion_r52920045>:
>
> > @@ -1,20 +0,0 @@
> > -object Test {
>
> I'll push a version in pending.
>
> What about just renaming b to something else?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lampepfl/dotty/pull/1082/files#r52920045>.
>



-- 
Martin Odersky
EPFL